Firmware/PlatformIO/main_mcu/src/bms.cpp

```diff
@@ -2,6 +2,14 @@
 
 #include <algorithm>
 
+// Consts for SoE calculation
+const float kDischargeCurrent = 45;
+const float kRegenCurrent = 4.2;
```

Comment:
```suggestion
const float kRegenCurrent = 45;
```
This should be the same as the max discharge current (for short periods)